functionary/ui/views/function.py

```diff
@@ -45,38 +45,46 @@ def get_queryset(self):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         function = self.object
-        env = function.package.environment
-        form = None
+        env = function.environment
 
         missing_variables = []
         if function.variables:
             all_vars = list(env.variables.values_list("name", flat=True))
             missing_variables = [
                 var for var in function.variables if var not in all_vars
             ]
+
         context["missing_variables"] = missing_variables
+
         if self.request.user.has_perm(Permission.TASK_CREATE, env):
-            form = TaskParameterForm(function)
+            context["form"] = TaskParameterForm(function)
 
-        context["form"] = form.render("forms/task_parameters.html") if form else None
```

Comment:
The explicit render isn't necessary so I cleaned that up here just to keep it consistent with the similar logic going on in the workflow task view.